### PR TITLE
fix: await filterAffected

### PR DESCRIPTION
**Pull request by Cammisuli**
With the latest version of Nx, the `filterAffected` function now returns a Promise.

calling `await filterAffected` will work with both the non-promise and promise based api.

### CODE REVIEW

**Comment by Cammisuli** [comment]
Ah dang, prettier auto formatted the other files here. Let me know if you want me to revert those formatting changes 🙂

File: packages/nx-semantic-release/src/common/git.ts [hunk 1]
```diff
@@ -33,7 +33,9 @@ export async function isCommitAffectingProjects({
 
   const affectedFiles = await listAffectedFilesInCommit(commit);
   const fileChanges = calculateFileChanges(affectedFiles, [], { projects });
-  const filteredGraph = filterAffected(graph, fileChanges);
+  const filteredGraph = await Promise.resolve(
+    filterAffected(graph, fileChanges)
+  );
```
Comment (TheUnderScorer) [inline packages/nx-semantic-release/src/common/git.ts:38]:
```suggestion
  const filteredGraph = await filterAffected(graph, fileChanges);
```

Wouldn't that work as well? If it won't return a promise, then `await` will just do nothing here and return the value

**Review by TheUnderScorer — COMMENTED** [review]
Thanks for the PR @Cammisuli! I left one comment.

**Pull request merged**